README.md

```diff
@@ -74,7 +74,7 @@ Make sure your Paloma Cosmos-SDK keys are stored and available on your environme
 
 Set the VALIDATOR env variable
 
-`export VALIDATOR="$(palomad keys list --list-names | head -n1)"`
+`export VALIDATOR="$(palomad keys list --output json | jq -r '.[0].address')"`
```

Comment:
variable VALIDATOR came from the paloma config where it's the name of the key in the keychain, so `VALIDATOR` so validator's address can't be used interchangeably in all places. Yes, it does work for some places, but not for all.

Reply:
Ah. I would be okay renaming it. `VALIDATOR_ADDRESS`? The fact remains sometimes `palomad keys list --list-names` lists nothing.